crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
+
+@is_command("Check all the crypto prices.")
+def crypto(message_dict):
+    user = message_dict["display-name"].lower()
+    message = [message_dict["message"]]
```

Comment:
Did you mean to assign this into a list..? It causes the regex on line 22 to except as you can't do a regex on a list.

Reply:
Ah, yes, I forgot to remove the array notation since I'm turning the string into an array with split.